src/applications/mhv-landing-page/components/NavCard.jsx

```diff
@@ -30,15 +30,16 @@ const NavCard = ({
         <span
           className={ariaLabel?.includes('unread') ? 'mhv-c-indicator' : ''}
         >
-          {text} {isExternal && externalLinkText}
+          {text}
+          {isExternal && ` ${externalLinkText}`}
+          {!isExternal && (
+            <va-icon
+              class="vads-u-margin-right--neg1 medium-screen:vads-u-margin-right--0"
+              icon="navigate_next"
+              size={4}
+            />
+          )}
```

Comment:
I think by moving the `<va-icon>` within the `<span>`, it breaks the small-screen layout where the chevron is right-aligned